frontend/src/component/project/Project/ProjectOverview.tsx

```diff
@@ -34,22 +35,62 @@ const StyledContentContainer = styled(Box)(() => ({
     minWidth: 0,
 }));
 
+const InfiniteProjectOverview = () => {
+    const projectId = useRequiredPathParam('projectId');
+    const { project, loading: projectLoading } = useProject(projectId, {
+        refreshInterval,
+    });
+    const [nextCursor, setNextCursor] = useState('');
+    const { features: searchFeatures, refetch, loading } = useFeatureSearch(nextCursor, projectId , { refreshInterval});
+    const { members, features, health, description, environments, stats } =
+        project;
+
+    return (
+        <StyledContainer>
+            <ProjectInfo
+                id={projectId}
+                description={description}
+                memberCount={members}
+                health={health}
+                features={features}
+                stats={stats}
+            />
+            <StyledContentContainer>
+                <ProjectStats stats={project.stats} />
+                <StyledProjectToggles>
+                    <ProjectFeatureToggles
```

Comment:
this is the same as previous code but now we can inject search features, search loading state and search refetch to refresh search after toggle flip